lib/acts_as_taggable_on_mongoid/models/concerns/tag_fields.rb

```diff
@@ -17,7 +17,9 @@ module TagFields
 
           # field :type, type: String
 
-          index({ name: 1, taggable_type: 1, context: 1 }, unique: true)
+          index({ name: 1, taggable_type: 1, context: 1, tagger_id: 1, tagger_type: 1 },
+                unique: true,
+                name:   "name_taggable_type_context_tagger")
```

Comment:
The breaking change.  With the introduction of taggers, we need to change this index.